vignettes/spatial_wwinference.Rmd

```diff
@@ -0,0 +1,2457 @@
+    as.data.frame() %>%
+    mutate(date = simulated_data_rng_corr_mat$hosp_data_eval$date) %>%
+    `colnames<-`(c(
+      paste0("Site ", 1:n_sites),
+      "Aux",
+      "date"
+    )) %>%
+    pivot_longer(cols = -date) %>%
+    `colnames<-`(c(
+      "date",
+      "subpop",
+      "value"
+    )) %>%
+    mutate(
+      model_type = "Rand. Corr. Matrix"
+    )
+) %>%
+  mutate(
+    model_type = factor(
+      model_type,
+      levels = c(
+        "IID",
+        "Exponential",
+        "Rand. Corr. Matrix"
+      )
+    )
+  )
+site_rt_data_plot <- ggplot(site_rt_data_full) +
+  geom_hline(
+    yintercept = 1,
+    color = "blue",
+    linetype = "dashed",
+    alpha = .7
+  ) +
+  geom_line(aes(
+    x = date,
+    y = (value)
+  )) +
+  facet_grid(subpop ~ model_type) +
+  scale_y_continuous(transform = "log10") +
+  xlab("") +
+  ylab("Subpopulation R(t)") +
+  theme_bw()
+
+# ------------------------------------------------------------------------------
+
+global_rt_data_plot / site_rt_data_plot +
+  plot_layout(heights = c(1, n_sites))
+```
+
+Similarly we can visualize the combined site level Rt curves.  Colors are the
+different sites.  Again, we have a blue horizontal line at Rt = 1 for
+reference of pathogen severity.
+
+```{r}
+site_rt_total_plot <- ggplot(site_rt_data_full) +
+  geom_hline(
+    yintercept = 1,
+    color = "blue",
+    linetype = "dashed",
+    alpha = .7
+  ) +
+  geom_line(aes(
+    x = date, y = value,
+    colour = factor(subpop)
+  )) +
+  facet_grid(~model_type,
+    scales = "fixed"
+  ) +
+  scale_color_manual(
+    breaks = c(
+      "Site 1", "Site 2", "Site 3", "Site 4", "Aux"
+    ),
+    values = c(
+      "darkviolet", "darkgreen",
+      "darkred", "darkorange", "grey"
+    )
+  ) +
+  scale_y_continuous(transform = "log10") +
+  xlab("") +
+  ylab("Subpopulation R(t)") +
+  labs(color = "Sites") +
+  theme_bw()
+
+site_rt_total_plot
+```
+
+
+### Visual of correlation matrix
+One other thing that we want to visualize, because it is the key difference
+between the data generation models, is the site to site correlation matrix.
+Here we do that, faceted by data generation model, we provide a scale that
+gives level of correlation.  We also make the area of each circle correspond
+with the correlation, so larger the area of the circle, larger the correlation.
+To remind ourselves, IID will have no correlations between wastewater sites.
+Exponential will have correlations between wastewater sites that are based off
+the exponential correlation function that uses the distances between wastewater
+treatment plants.  Rand. Corr. Matrix will have correlations between wastewater
+treatment plants that are based off some unknown factors, to emulate this we
+just set random correlations between sites.
+
+<div class="fold s">
+```{r fig.height=5, fig.width=10}
+# Correlation Visuals ----------------------------------------------------------
+corr_matrix_df <- rbind(
+  simulated_data_iid$corr_matrix %>%
```

Comment:
This actually would be a lovely plot function for the package, but def think we can make it a separate issue.

Reply:
I have a controversial opinion that I like vignettes that at least in one instance show you how to plot the data without a wrapper function (though I think here since it is repetitive and verbose, there is an argument for eventually putting these plots in wrapper functions)

Reply:
That's not too controversial IMO. But yeah here it's repetitive so I think simplifying it is worth it